fml/trace_event.cc

```diff
@@ -22,6 +23,27 @@ namespace tracing {
 
 #if TIMELINE_ENABLED
 
+namespace {
+AsciiTrie gWhitelist;
```

Comment:
Can we ifdef this out of release builds, and not do any whitelisting for release?

Reply:
It is inside of one already "TIMELINE_ENABLED"  I'm not sure if that's one release builds uses.